src/others/xml/lv_xml_style.c

```diff
@@ -0,0 +1,202 @@
+/**
+ * @file lv_xml_style.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../../lvgl.h"
+#if LV_USE_XML
+
+#include "lv_xml_base_types.h"
+#include "lv_xml_parser.h"
+#include "lv_xml_style.h"
+#include "lv_xml_utils.h"
+#include <string.h>
```

Comment:
Probably just leftover
```suggestion
```

Reply:
Not really the xml style implementations use some functions from the string module, removing it causes a warning of implicit function definition which may be treated as error by the CI or other plaform that enforces warning as error.

For now I suggest to keep on its place, and later we can revisit a better place to put this header.